parser.py

```diff
@@ -0,0 +1,45 @@
+# Author: Pankil Shah
+# Created on: 03/13/2021
+# Description: Parse a corpus and dump the sentences into the database 
+
+from nltk.corpus import brown
+import nltk.data
+import logic
+import datetime
+from os import listdir
+from os.path import isfile, join
+
+#print(brown.tagged_sents()[0:2])
+
+#dirPath = 'E:\\SLU\\Sem1\\Principles of SD\\Project Material\\brown'
+path = input("Enter path of the corpus directory: ")
+language = input("Enter language: ")
+#language = "english"
+
+# list of all file names in the corpus directory
+fileNameList = [f for f in listdir(path) if isfile(join(path, f))]
+languageId = str(logic.SQLQuery("select Lang_ID from lang_ref where Lang_Desc = '" + language + "'")[0])
```

Comment:
If it possible we should change all of the string concatenation to using f-string. It makes the code more readable and we don't have to use +. The backend file has some example, here is one. 

language_selected = request.form.get('language')

Lang_ID = logic.SQLQuery(f"SELECT Lang_ID FROM Lang_Ref WHERE Lang_Desc = '{language_selected}';")

Reply:
I think this should be left to the person who writes the code because I am more comfortable with using concatenations.